### ISSUE TITLE
Enum default value is not consistent with other field types

### ISSUE THREAD

**Issue by davidharrigan** [facebookincubator/ent#644]
Version: v0.2.6

Given these two fields:
```golang
return []ent.Field{
	field.String("strfield").
		Default("default"),

	field.Enum("enumfield").
		Values("default", "something_else").
		Default("default"),
}
```
The first one works as expected where the default value is filled in during creation and during migration filling the default value for the existing rows.

The second one does not work the same way, even though at least for postgres they are the same varchar type underneath.  The default value is filled in during creation, but not during migration on the existing rows.  I'm looking for clarity if this is intentional.  If so, is there a recommended way of providing default values on existing rows using ent?

Note that if the above code is provided from a Mixin, the generated code fails due to an "unused variable" error from the runtime package.  Not sure if this is a separate issue or stems from the same issue.

**Comment by a8m** [comment]
Hey @davidharrigan and thanks for reporting this issue.

> The second one does not work the same way, even though at least for postgres they are the same varchar type underneath. 

Yes, it's too complicated to manage enum types in PostgreSQL and we decided to use `varchar` at the moment in the database, and enforce this in the application-level only.

>  The default value is filled in during creation, but not during migration on the existing rows. 

> Note that if the above code is provided from a Mixin, the generated code fails due to an "unused variable" error from the runtime package. Not sure if this is a separate issue or stems from the same issue.

Checking this now and I'll update this issue

**Comment by a8m** [comment]
I wasn't able to reproduce the Mixin issue on both v0.2.6 and master. See branch: [enum-issue](https://github.com/facebookincubator/ent/tree/enum-issue).

**Comment by a8m** [comment]
Hey @davidharrigan, issue resolved with #646.
Thank you for reporting this issue 😃

**Comment by davidharrigan** [comment]
thank you for the speedy fix!
